projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.7-beta - 2022-02-22
+### Enhancements
+- Dashboard: improve loading of plugins page.
+- Dashboard: remove marketplace related add plugins menu redirect.
+- Subscribe block: change border color automatically when button background color changes.
+- Subscribe block: several updates including name change, display of current subscribers, and a new style option.
+
+### Improved compatibility
+- Performance: improve performance for sites without VideoPress enabled.
```

Comment:
Should we mention that this improves the performance of the Jetpack dashboard specifically?